index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Mintbean Hackathon - Noisemaker</title>
+    <script src="https://unpkg.com/tone"></script>
+    <script type="module" src="./scripts/audio.js"></script>
```

Comment:
Props for using vanilla javascript! My feedback here would be that it might be nice to add a bit of styling, or text indicating what song is playing.